scripts/local_runtimes_common_integration_test.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+
+# Copyright 2017 Google Inc. All rights reserved.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# exit on command failure
+set -e
+
+readonly dir=$(dirname $0)
+readonly projectRoot="$dir/.."
+readonly testAppDir="$projectRoot/test-application"
+readonly deployDir="$testAppDir/target/deploy"
+
+APP_IMAGE='openjdk-local-integration'
+CONTAINER=${APP_IMAGE}-container
+OUTPUT_FILE=${CONTAINER}-output.txt
+DEPLOYMENT_TOKEN=$(uuidgen)
+
+readonly imageUnderTest=$1
+if [[ -z "$imageUnderTest" ]]; then
+  echo "Usage: ${0} <image_under_test>"
+  exit 1
+fi
+
+if [[ -z ${GOOGLE_APPLICATION_CREDENTIALS} ]]; then
```

Comment:
The readme instructions don't mention `GOOGLE_APPLICATION_CREDENTIALS`.
Can we just leverage gcloud for the default credentials?